### PR TITLE
[Fixes #279] Remove RuntimeInfo middleware, extensions and tests

### LINKED ISSUE #279: Remove UseRuntimeInfoPage()
What little is left in the current implementation is simply not useful. Let's remove it for now. And bring it back later.

Following are the locations where this middleware is currently being used:
## MusicStore repo
https://github.com/aspnet/MusicStore/blob/1d19518c53ac2a29d8ccf716a23c2cc06fc8e13c/src/MusicStore/StartupNtlmAuthentication.cs#L109
https://github.com/aspnet/MusicStore/blob/6af8da4713d52c25f7f0f6fa9ba6abd20f687473/src/MusicStore/Startup.cs#L114
https://github.com/aspnet/MusicStore/blob/6af8da4713d52c25f7f0f6fa9ba6abd20f687473/src/MusicStore/StartupOpenIdConnect.cs#L121
https://github.com/aspnet/MusicStore/blob/6af8da4713d52c25f7f0f6fa9ba6abd20f687473/shared/Mocks/StartupSocialTesting.cs#L114
https://github.com/aspnet/MusicStore/blob/1d19518c53ac2a29d8ccf716a23c2cc06fc8e13c/src/MusicStore/StartupNtlmAuthentication.cs#L109

### PR DESCRIPTION
**Pull request by kichalla**
@Eilon 

Will create a bug in Docs repo to remove its references
https://docs.asp.net/en/latest/fundamentals/diagnostics.html?#the-runtime-info-page

### CODE REVIEW

File: samples/RuntimeInfoPageSample/Startup.cs [hunk 1]
```diff
@@ -8,7 +8,8 @@ public class Startup
     {
         public void Configure(IApplicationBuilder app)
         {
-            app.UseRuntimeInfoPage();
+            // Temporarily disabling this https://github.com/aspnet/Diagnostics/issues/279
+            // app.UseRuntimeInfoPage();
```
Comment (Eilon) [inline samples/RuntimeInfoPageSample/Startup.cs]: Why not delete the whole sample?

File: src/Microsoft.AspNetCore.Diagnostics/RuntimeInfo/RuntimeInfoExtensions.cs [hunk 2]
```diff
@@ -8,7 +8,8 @@
 
 namespace Microsoft.AspNetCore.Builder
 {
-    public static class RuntimeInfoExtensions
+    // Temporarily disabling these extensions https://github.com/aspnet/Diagnostics/issues/279
+    static class RuntimeInfoExtensions
```
Comment (Eilon) [inline src/Microsoft.AspNetCore.Diagnostics/RuntimeInfo/RuntimeInfoExtensions.cs]: I'm confused - why not delete all the code?

**Comment by Eilon** [comment]
🕐

**Comment by kichalla** [comment]
:up: :date: I found references to this middleware in MusicStore which I cleaned and pushed.

File: src/Microsoft.AspNetCore.Diagnostics.Elm/project.json [hunk 3]
```diff
@@ -7,7 +7,12 @@
     "nowarn": [
       "CS1591"
     ],
-    "xmlDoc": true
+    "xmlDoc": true,
+    "compile": {
+      "exclude": [
+        "Views/*.cshtml"
```
Comment (Eilon) [inline src/Microsoft.AspNetCore.Diagnostics.Elm/project.json]: Hmm why is this needed? cshtml files aren't compiled by default...

Comment (kichalla) [inline src/Microsoft.AspNetCore.Diagnostics.Elm/project.json]: Good point.

**Comment by Eilon** [comment]
BTW please make sure you scan all other repos and first remove all references to the RuntimeInfo page, and then commit this change.

So :shipit: after that.

**Comment by kichalla** [comment]
af19899927516718bdc05507612dcc17901fb937
